src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/PlacementRequestService.kt

```diff
@@ -75,11 +76,18 @@ class PlacementRequestService(
 
   fun getAllReallocatable(
     page: Int?,
+    sortField: TaskSortField,
     sortDirection: SortDirection?,
     allocatedFilter: AllocatedFilter?,
   ): Pair<List<PlacementRequestEntity>, PaginationMetadata?> {
-    val sortField = "created_at"
-    val pageable = getPageable(sortField, sortDirection, page)
+    val pageable = getPageable(
+      // Convert to snake_case, because the findAllReallocatable* methods are native SQL queries
+      when (sortField) {
+        TaskSortField.createdAt -> "created_at"
+      },
```

Comment:
Optional, but this could be replaced with `sortField.value` if the `TaskSortField` enum is defined in the spec to use snake case for its acceptable values.